win32ss/drivers/miniport/xboxvmp/xboxvmp.h

```diff
@@ -11,42 +11,26 @@
 
 /* INCLUDES *******************************************************************/
 
+/*
+ * FIXME: specify headers properly in the triangle brackets and rearrange them
+ * in a way so it would be simpler to add NDK and other headers for debugging.
+ */
```

Comment:
Do you want to fix that in a separate PR (together with further refactoring)?

Reply:
Yes, this is for later.